domains/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -1,32 +1,31 @@
 //! # Ethereum Beacon Client
 #![cfg_attr(not(feature = "std"), no_std)]
+#![allow(clippy::large_enum_variant)] // Runtime-generated enums
```

Comment:
This is happening due to three variants of `pallet::call` enum generated by the macro differs in size. Reason being that the bridge blocking call do not take any argument while other two call's arguments are pretty big. 

Since the enum is generated by runtime, I do not think I can suppress it for that specific structure.

Reply:
But you can box large enum variant contents to make it not be as large anymore. We have it in a few pallets, you can just search for `Box` in them and use as an example.